javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,22 @@
+/* exported filterOutStrings */
+function filterOutStrings(values) {
+
+  var noStrings = [];
+
+  for (var i = 0; i < values.length; i++) {
+
+    if (typeof values[i] === 'number') {
```

Comment:
This function is meant to filter out strings, not collect all numbers. What if the input array included `null` or `undefined`?

Reply:
<img width="1920" alt="Screen Shot 2022-03-29 at 22 38 52" src="https://user-images.githubusercontent.com/99709530/160759134-bee899cf-6325-46ab-9962-79f4cced95ad.png">